packages/roosterjs-editor-api/lib/utils/execCommand.ts

```diff
@@ -45,3 +45,19 @@ function isPendableFormatCommand(command: DocumentCommand): boolean {
     }
     return pendableFormatCommands.indexOf(command) >= 0;
 }
+
+function setPendableFormatState(pendableFormats: PendableFormatState, command: DocumentCommand) {
+    let keys = Object.keys(PendableFormatCommandMap) as PendableFormatNames[];
+    return keys.reduce((state, key) => {
+        state[key] =
+            pendableFormatKeyToCommand(key) === command
+                ? !pendableFormats[key]
+                : pendableFormats[key];
+        return state;
+    }, <PendableFormatState>{});
+}
+
+function pendableFormatKeyToCommand(key: string) {
+    let newKey = key.replace('is', '');

```

Comment:
Do not do this string operation. You can use PendableFormatCommandMap to convert between command and style name.